src/VisualStudio/Core/Def/Implementation/AbstractOleCommandTarget.Query.cs

```diff
@@ -120,6 +120,9 @@ private int QueryRoslynGroupStatus(ref Guid pguidCmdGroup, uint commandCount, OL
         {
             switch (prgCmds[0].cmdID)
             {
+                case ID.RoslynCommands.GoToBase:
```

Comment:
this is also not needed. To define and handle a new command in modern editor commanding you need:
1. Definition in vsct
2. Guid/id constants matching definition in vsct
3. new CommandArgs type
4. CommandBindingDefinition export that binds guid/id to that commandargs type.
5. CommandHanlder export that handles your commandargs

You can do it all in Roslyn, or steps 1-4 can be done in the editor, then the command can be reused by other VS components